project-base/app/config/packages/gopay.yaml

```diff
@@ -1,15 +1,10 @@
 parameters:
     gopay_config:
-        en:
-            goid: '%env(GOPAY_EN_GOID)%'
-            clientId: '%env(GOPAY_EN_CLIENTID)%'
-            clientSecret: '%env(GOPAY_EN_CLIENTSECRET)%'
-            language: EN
-        cs:
-            goid: '%env(GOPAY_CS_GOID)%'
-            clientId: '%env(GOPAY_CS_CLIENTID)%'
-            clientSecret: '%env(GOPAY_CS_CLIENTSECRET)%'
-            language: CS
+        #        en:
+        #            goid: '%env(GOPAY_EN_GOID)%'
+        #            clientId: '%env(GOPAY_EN_CLIENTID)%'
+        #            clientSecret: '%env(GOPAY_EN_CLIENTSECRET)%'
+        #            language: EN
```

Comment:
With this configuration commented, we will not have Gopay enabled on our devel/production server, where the Gopay keys for both cs and en domains are set.

I would like to keep the env variables and gopay config as it is, and just check whether the values are set or empty to determine whether the Gopay request should be made or not.

Reply:
Ok, I have added it back